### DISCUSSION TITLE
Network sluggish and hangs. Reboot required.

### DISCUSSION THREAD

**Discussion by SergioRius** [Koenkk/zigbee2mqtt#13975] [General]
I'm having this problem in my network. I hope someone can give me a hand.
My network it's only composed by Aqara wireless wall switches (6x WXKG07LM), Inrr downlights (12x (RS 227 T), and recently added come motion (3x IH012-RT01) and illuminance sensors (3x GZCGQ01LM).

I have controllerx configured to pair switches actions to lights (simple turn on atm) and automoli for turning on lights with pir. Then I have a node-red flow for circadian lighting calculation that updates brightness and temp when the lights are on. But I doubt all of this have something to do with the problem.

The lights stop responding. So I took a look at the logs and it seems like all mqtt orders are processing except lights. And it goes slow.
When I try to restart the z2m service, it takes ages and it seems to fail and stay on.
If I reboot the VM, it hangs and I have to force it stop from the hipervisor.

Then in the logs I periodically see this actions for all the illumination sensors:
```
info  2022-09-11 22:57:30: Configuring 'dormezra_il'
...
error 2022-09-11 22:57:30: Failed to configure 'dormezra_il', attempt 1 (Error: Bind 0x54ef4410001ffea4/1 msIlluminanceMeasurement from '0x9035eafffe51bb8b/1' failed (ZdoRequest error)
    at Driver.zdoRequest (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/driver.ts:530:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at Object.func (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/ezsp/adapter/ezspAdapter.ts:492:13)
    at Queue.executeNext (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32))
```

Does anyone know how to debug and fix this? It's something with the devices? The coordinator?

**Comment by DubhAd** [comment]
What coordinator are you using? From the log snippet I'm guessing something EZSP?

**Reply by SergioRius** [reply]
> What coordinator are you using? From the log snippet I'm guessing something EZSP?

Bingo! I'm using a remote wifi sonoff zbridge reflashed.
Unfortunately the services all run in my homelab, that is in the basement. The floorings (Don't know how is it said in english) are so heavy. More than half meter in thickness. So I have to use the wifi infraestructure.

Could the problem be the coordinator?

**Reply by DubhAd** [reply]
It could be, the ZBBridge is not a great solution. You'd be better off with a wired adapter than a WiFi one, and a lot better off with something CC2652 based either way.

**Reply by SergioRius** [reply]
> It could be, the ZBBridge is not a great solution. You'd be better off with a wired adapter than a WiFi one, and a lot better off with something CC2652 based either way.

What is recommended, nowadays? I guess ZZH! ?
What about the zigstar one? I don't see any references on the guide about it.

Update: I've ordered the Zigstar Lan version. We'll see.
